updater/lib/dependabot/job.rb

```diff
@@ -171,7 +171,14 @@ def reject_external_code?
     # separately, if required.
     #
     # rubocop:disable Metrics/PerceivedComplexity
+    # rubocop:disable Metrics/CyclomaticComplexity
     def allowed_update?(dependency)
+      # Ignoring all versions is another way to say no updates allowed
+      if completely_ignored?(dependency) && !security_updates_only?
```

Comment:
should this condition be in the `completely_ignored` method? Also, for my curiosity, any reason why security_updates have to be false?

Finally; any correlation between this and the ongoing alert rules by @landongrindheim and @bdragon ?

Reply:
I had the same thought as @abdulapopoola, but also `ignore_conditions_for` method is already passing around the `security_updates_only?` flag, so maybe whatever the correct behavior for security updates is, it is already being considered?

Reply:
Ah, didn't notice it was passed in `ignore_conditions_for` already, thanks @deivid-rodriguez I'll remove it.

Reply:
For alert rules, we shouldn't even get in the job, it will ignore before creating the job itself.